web/src/app/environments/view/private/environments-view-private.component.ts

```diff
@@ -17,6 +19,7 @@ export class EnvironmentsViewPrivateComponent implements OnInit {
     private route: ActivatedRoute,
     private router: Router,
     private environmentService: EnvironmentService,
+    private dialog: MatDialog,
```

Comment:
For consistency this should be above the `EnvironmentService` and below `Router`

Reply:
I will merge this as is and I will reorder these in my next PR 🤓